packages/inter-protocol/src/proposals/addAssetToVault.js

```diff
@@ -216,8 +220,13 @@ export const registerScaledPriceAuthority = async (
     10n ** BigInt(decimalPlacesRun),
     runBrand,
   );
-  const initialPrice = initialPricePct
-    ? makeRatio(BigInt(initialPricePct), interchainBrand, 100n, runBrand)
+  const initialPrice = initialPriceRaw
+    ? makeRatio(
+        BigInt(initialPriceRaw * 10_000),
```

Comment:
Document that this throws `RangeError` given `0.000000005`? Or add `Math.round()`?

Reply:
looks like we have `parseRatio` in `packages/zoe/src/contractSupport/ratio.js` that should work here.